src/Yoti.Auth/Constants/Api.cs

```diff
@@ -9,7 +9,7 @@ public static class Api
         public const string YotiApiPathPrefix = "api/v1";
         public readonly static string DefaultYotiApiUrl = string.Join("/", DefaultYotiHost, YotiApiPathPrefix);
 
-        public const string YotiDocsPathPrefix = "idverify/v1";
+        public const string YotiDocsPathPrefix = "idverify/v1/";
```

Comment:
Should this have a trailing slash? (it differs to _YotiApiPathPrefix_  `api/v1`)

Reply:
I left a note about that in the PR description - the last bullet 👍